src/main/java/frc/robot/commands/ClimbPrepSequence.java

```diff
@@ -12,19 +12,22 @@
 import frc.robot.subsystems.shooter.Shooter.ShooterMode;
 
 /**
- * gets the robot ready to climb by moving the intake, elevator, and shooter to their appropriate
- * positions
+ * Gets the robot ready to climb by moving the intake, elevator, and shooter to their appropriate
+ * positions. Does not wait until everything is at their desired positions!
  */
-public class PrepClimbSequence extends SequentialCommandGroup {
-  public PrepClimbSequence(Intake intake, Elevator elevator, Shooter shooter) {
+public class ClimbPrepSequence extends SequentialCommandGroup {
+  public ClimbPrepSequence(Intake intake, Elevator elevator, Shooter shooter) {
     addRequirements(intake, elevator, shooter);
+    SequentialCommandGroup waitTillSafe =
+        new SequentialCommandGroup(
+            new WaitUntilCommand(shooter::clearOfConveyorZone),
+            new WaitUntilCommand(intake::clearOfConveyorZone));
```

Comment:
I don't know if this is quite necessary to be it's own commandgroup. It works, of course, but it's pretty clear what's happening from the two `WaitUntilCommand`s (and we don't do this anywhere else - we should always do it this way or always not).

Reply:
Is it worth changing?

Reply:
I feel like this one is borderline but probably fine either way

Reply:
Fine either way so let's leave it.